qutebrowser/browser/hints.py

```diff
@@ -77,13 +78,21 @@ class HintLabel(QLabel):
     """
 
     def __init__(self, elem, context):
-        super().__init__(parent=context.tab)
-        self._context = context
-        self.elem = elem
+        super().__init__()
+        self._reusable_init(elem, context)
 
+        # Those attributes are common for all HintLabel objects, and they only
+        # need to be initialized once per object
         self.setAttribute(Qt.WA_StyledBackground, True)
```

Comment:
We need to double check there aren't any hint settings that won't get updated automatically (ie: that need changes to this code). From basic testing though, I don't think that's needed.

Reply:
I don't think there is any. The `STYLESHEET` property is constant.